index.html

```diff
@@ -2259,12 +2294,13 @@ <h4 id="cryptosuiteString">The `cryptosuiteString` Datatype</h3>
           <h4 id="multibase">The `multibase` Datatype</h3>
 
           <p>
-[[?MULTIBASE]]-encoded strings are used to encode binary data into ASCII-only
-formats, which are useful in environments that cannot directly represent binary
-values. This specification makes use of this encoding. In environments that
-support data types for string values, such as RDF [[?RDF-CONCEPTS]],
-[[?MULTIBASE]]-encoded content is indicated using a literal value whose datatype
-is set to `https://w3id.org/security#multibase`.
+<a href="#multibase">Multibase</a>-encoded strings are used to encode binary
+data into ASCII-only formats, which are useful in environments that cannot
+directly represent binary values. This specification makes use of this encoding.
+In environments that support data types for string values, such as RDF
+[[?RDF-CONCEPTS]], <a href="#multibase">Multibase</a>-encoded content is
```

Comment:
If we're citing RDF-CONCEPTS, it would be useful to have a sentence with the citation describing how it informs implementations in this case.

Reply:
It is not clear what sort of sentence you want written here. Please provide concrete specification text, or at least more prose suggesting what you want to see here.